### ISSUE TITLE
[A11y] Desktop/Mobile Collection Landing: Link Names for images

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1747]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Collection Show

### Admin user?

- [ ] Yes

### Issue Description

Issue on Collection Landing Page, image thumbnails
https://staging.oregondigital.org/collections/gwilliams?locale=en

Failed Instance
link-name: Ensures links have discernible text

Image 0
Path: `#image-0 > a`
Snippet:
`<a href="/concern/images/df66v729g?locale=en"> <img alt="" src="/downloads/w0892992w?file=thumbnail"> <h3 class="hover-over-collection-text show-me0"> Ashland-Klamath Falls Highway </h3> </a>`

Image 1
Path: `#image-1 > a`
Snippet: 
`<a href="/concern/images/df66v7022?locale=en"> <img alt="" src="/downloads/5d86p0224?file=thumbnail"> <h3 class="hover-over-collection-text show-me1"> Falls from above </h3> </a>`

Image 2
Path: `#image-2 > a`
Snippet:
`<a href="/concern/images/df66vb95p?locale=en"> <img alt="" src="/downloads/hx11xf24j?file=thumbnail"> <h3 class="hover-over-collection-text show-me2"> Drinking mineral water </h3> </a>`

Image 3
Path: `#image-3 > a`
Snippet:
`<a href="/concern/images/df66v695v?locale=en"> <img alt="" src="/downloads/8336h188j?file=thumbnail"> <h3 class="hover-over-collection-text show-me3"> Avalanche Falls </h3> </a>`

### Suggested Fix

Fix the following:

- Element is in tab order and does not have accessible text

Fix ONE of the following:

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

**Comment by CGillen** [comment]
Fixed by #1950
